### ISSUE TITLE
See process environment

### ISSUE THREAD

**Issue by scottcmarks** [dalance/procs#143]
`ps ewwp $(pgrep -i emacs)` gives the process (startup) environment (for `Emacs` in this example), although since the entries are separated by space characters, it's less useful when the values contain spaces.  The same idea, but with a nice separator, would be **very** welcome.

**Comment by dalance** [comment]
I added `Env` column.
The current format is like below:

```
MAKEOPTS="-j8" ZPLUG_FILTER="fzf-tmux:fzf:peco:percol:fzy:zaw" ZPLUG_USE_CACHE="true"
```

**Comment by dalance** [comment]
I released v0.14.0.
